test/sqllogictest/pg_catalog_class.slt

```diff
@@ -32,3 +32,17 @@ FROM pg_catalog.pg_class
 WHERE relname = (SELECT name FROM mz_indexes WHERE on_id = (SELECT id FROM mz_objects WHERE name = 'a'));
 ----
 0 0 0 0 false p i 0 false false false false d false false
+
+statement ok
+SET DATABASE alpha;
+CREATE TABLE a (b int);
+SET DATABASE beta;
+
+# Normal table
+query TIIIIBTTIBBBBTBB
+SELECT relname, reloftype, relam, reltablespace, reltoastrelid, relhasindex, relpersistence, relkind, relchecks,
```

Comment:
I'm not clear on the goal if this query. It has `relname = 'a'` but then the result has relname as `q`.

Reply:
It's not clear to me how to express an _empty_ expected result, so my goal with this is to get a failure that tells me how to do it ;-)

Reply:
You know how to run these locally, yeah? https://github.com/MaterializeInc/materialize/blob/main/doc/developer/sqllogictest.md

You'll almost certainly want to try out the `--rewrite-results` option too!

Reply:
Thanks!